### PR TITLE
build: bump version to run sha256 for background job arguments migration

### PR DESCRIPTION
**Pull request by kesselb**
* Resolves: error when enabling an app

## Summary

Follow-up for https://github.com/nextcloud/server/pull/47769

## TODO

- [ ] CI

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]

**Comment by kesselb** [comment]
I pushed another commit to bump the 3rdparty version and fix the check "Block merging with outdated 3rdparty".

**Review by susnux — APPROVED** [review]

File: 3rdparty
Comment (susnux) [inline 3rdparty:1]: This is unrelated no?

**Comment by AndyScherzinger** [comment]
@artonge is this a must merge for v30.0.0?

**Comment by susnux** [comment]
> is this a must merge for v30.0.0?

The version is bumped during the RC creation by the release team, so this should be good for all stable branches.
But for current master we did not bump the version so it is only requited for 31.

File: 3rdparty
Comment (kesselb) [reply 3rdparty:1]: > I pushed another commit to bump the 3rdparty version and fix the check "Block merging with outdated 3rdparty".

It's unrelated, but without the required check for 3rdparty is failing. If you prefer, I drop the commit and send another pr.

Comment (AndyScherzinger) [reply 3rdparty:1]: Merged, done is better than perfect (PRs)

Comment (susnux) [reply 3rdparty:1]: No makes sense, so :+1:
